### ISSUE TITLE
sem-apply --set parameter cannot be repeated

### ISSUE THREAD

**Issue by pelder-lumenix** [mbryzek/schema-evolution-manager#68]
I tried multiple psql parameters in a script:

```
create role user_ro login password :'ro_pwd';
create role user_rw login password :'rw_pwd';
```

When I run `sem-apply` with multiple `--set`:

```sem-apply --url xxx --set ro_pwd=abc --set rw_pwd=def```

I get script errors. Running the script directly with psql does not have those errors. On investigation, only one of the variables gets passed to psql. (The last variable wins.)

**Comment by pelder-lumenix** [comment]
@mbryzek : Tested your changes for #70. Everything works as I expect. Thank you.

**Comment by pelder-lumenix** [comment]
Fixed by #70
